src/storage/object.ts

```diff
@@ -721,10 +721,10 @@ export class ObjectStorage {
       isUpsert: options?.upsert ?? false,
     })
 
-    const { secret: jwtSecret } = await getJwtSecret(this.db.tenantId)
+    const { urlSigningKey } = await getJwtSecret(this.db.tenantId)
```

Comment:
is the `urlSigningKey` always available? even for projects that do not have the jwks generated?
If not, shall we keep this backwards compatible

Reply:
Yes, it is always available. It is always what should be used for signing URLs. If there is no JWK for url signing it is the old jwt secret string.